docs/drivers.rst

```diff
@@ -1,252 +1,32 @@
-.. _adafruit-libndrivers:
-
-Additional Adafruit Libraries and Drivers on GitHub
-===================================================
+Additional CircuitPython Libraries and Drivers on GitHub
+=========================================================
 
 These are libraries and drivers available in separate GitHub repos. They are
 designed for use with CircuitPython and may or may not work with
 `MicroPython <https://micropython.org>`_.
 
-.. _bundle_installation:
 
-Bundle
-------
+Adafruit CircuitPython Bundle
```

Comment:
I would include `Library` in this for `Adafruit CircuitPython Library Bundle`. Better to be as clear as possible.